ui/app/src/components/DeleteDialog/DeleteDialogUIBody.tsx

```diff
@@ -141,12 +142,20 @@ export function DeleteDialogUIBody(props: DeleteDialogContentUIProps) {
 			</Grid>
 			<Grid size={{ xs: 12, sm: 5 }}>
 				<form noValidate autoComplete="off">
+					<TextField
+						label={<FormattedMessage defaultMessage="Title" />}
+						autoFocus
+						fullWidth
+						sx={{ mb: 2 }}
+						value={title}
+						onChange={(e) => onInputChange(e, 'title')}
+					/>
 					<TextFieldWithMax
 						label={<FormattedMessage id="deleteDialog.submissionCommentLabel" defaultMessage="Submission Comment" />}
 						multiline
 						value={comment}
-						onChange={onCommentChange}
-						required={isCommentRequired}
```

Comment:
Did we remove the site config setting for delete comment to be required? Service fails with no comment?